shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/mysql/BaseMySQLScalingIT.java

```diff
@@ -46,15 +48,13 @@ public BaseMySQLScalingIT() {
     @SneakyThrows
     protected void initScalingEnvironment() {
         try (Connection connection = getProxyDataSource("").getConnection()) {
-            connection.createStatement().execute(getCommonSQLCommand().getCreateDatabase());
-            connection.createStatement().execute(getExtraSQLCommand().getUseDatabase());
-            String addResource = getExtraSQLCommand().getAddResource();
-            for (String source : getSourceDataSourceNames()) {
-                connection.createStatement().execute(String.format(addResource, source, getDatabaseUrl(), source));
-            }
-            for (String target : getTargetDataSourceNames()) {
-                connection.createStatement().execute(String.format(addResource, target, getDatabaseUrl(), target));
-            }
+            connection.createStatement().execute("CREATE DATABASE sharding_db");
+            connection.createStatement().execute("USE sharding_db");
+            connection.createStatement().execute(String.format(ADD_RESOURCE_SQL, "ds_0", getDatabaseIpAndPort(), "ds_0"));
+            connection.createStatement().execute(String.format(ADD_RESOURCE_SQL, "ds_1", getDatabaseIpAndPort(), "ds_1"));
+            connection.createStatement().execute(String.format(ADD_RESOURCE_SQL, "ds_2", getDatabaseIpAndPort(), "ds_2"));
+            connection.createStatement().execute(String.format(ADD_RESOURCE_SQL, "ds_3", getDatabaseIpAndPort(), "ds_3"));
+            connection.createStatement().execute(String.format(ADD_RESOURCE_SQL, "ds_4", getDatabaseIpAndPort(), "ds_4"));
```

Comment:
Could we encapsulate the common code of a case into one class, so it could be reused by any databases, e.g. MySQL and PostgreSQL etc.